src/function/scalar/string/like.cpp

```diff
@@ -70,4 +119,10 @@ void LikeFun::RegisterFunction(BuiltinFunctions &set) {
 	                               ScalarFunction::BinaryFunction<string_t, string_t, bool, NotLikeOperator, true>));
 }
 
+void LikeEscapeFun::RegisterFunction(BuiltinFunctions &set) {
+	set.AddFunction({"like_escape"}, ScalarFunction({SQLType::VARCHAR, SQLType::VARCHAR, SQLType::VARCHAR},
+	                                                SQLType::BOOLEAN, like_escape_function<LikeEscapeOperator>));
+	set.AddFunction({"!like_escape"}, ScalarFunction({SQLType::VARCHAR, SQLType::VARCHAR, SQLType::VARCHAR},
```

Comment:
Call the function not_like_escape, otherwise the function cannot be written in normal SQL.